pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -128,9 +128,9 @@ const VerifyPage = ({
     <div>
       {!account && (
         <div>
-          <a className={styles.connect} onClick={connectToStarknet}>
-            connect your Starknet wallet
-          </a>
+          <button className={styles.connect}>
+            <a onClick={connectToStarknet}>Connect your Starknet wallet</a>
```

Comment:
Can you remove the <a> tag please ? And put the onclick in the button ? So that the underline disappears